integrations/dbt/bigquery/setup.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+from setuptools import find_packages
+from setuptools import setup
+
+package_name = "dbt-openlineage-bigquery"
```

Comment:
We'll want to use the package name `openlineage-dbt-bigquery `

Reply:
Done.